test/runtests.jl

```diff
@@ -2,30 +2,26 @@ using WaterModels
 import InfrastructureModels
 import Memento
 
+# Suppress warnings during testing.
+Memento.setlevel!(Memento.getlogger(InfrastructureModels), "error")
+WaterModels.logger_config!("error")
+
+# For checking status codes
+import MathOptInterface
+const MOI = MathOptInterface
```

Comment:
You should be able to drop these in the tests if you prefer becouse WaterModels exports the MOI status codes.  This is a style choice each package can make, i.e. If the tests should emulate a user calling `using` or a package developer calling `import`.